torch/nn/utils/parametrize.py

```diff
@@ -7,6 +7,9 @@
 from contextlib import contextmanager
 from typing import Union, Optional, Dict, Tuple, Sequence
 
+__all__ = ['cached', 'ParametrizationList', 'register_parametrization', 'is_parametrized', 'remove_parametrizations',
```

Comment:
It kinda feels like `cached` should be an internal implementation detail to me, although I guess it has de facto been public so this is the right thing to do here. Maybe we want to move this to private eventually? @albanD @Lezcano

Reply:
`cached` is documented and part of the public API.
I am not sure about 'type_before_parametrizations', 'transfer_parametrizations_and_params' though, as they are not documented.
cc @albanD